src/lib/cid.js

```diff
@@ -1,18 +1,47 @@
-import CID from 'cids'
+import { CID } from 'multiformats'
+import * as dagCbor from '@ipld/dag-cbor'
+import * as dagPb from '@ipld/dag-pb'
```

Comment:
without knowing what this file is meant to do, you could also add in some more codecs to make it able to detect a wider range, here's an example that pulls in dag-json, raw (very common) and json: https://github.com/ipld/js-car/blob/master/examples/verify-car.js#L11-L15, you can see a code->codec lookup table in there for this as well. The same pattern is currently used in js-ipfs too.

Reply:
oh, and the js bitcoin codec is working again now if you want to add it back in; there's actually 3 codecs for it and they have the same interface as the other codecs so you can just plug them in to the same lookup table, docs @ https://github.com/ipld/js-bitcoin

Reply:
We'll have to save that for another day. This PR is part of a large number of updates blocking webui's support of CAR files. see ipfs/ipfs-webui#1965

Reply:
Opened https://github.com/ipfs/ipld-explorer-components/issues/335